packages/generator/src/artifacts/preact-jsx/pattern.ts

```diff
@@ -4,14 +4,20 @@ import { match } from 'ts-pattern'
 import type { Context } from '../../engines'
 
 export function generatePreactJsxPattern(ctx: Context, filters?: ArtifactFilters) {
-  const { typeName, factoryName } = ctx.jsx
+  const { typeName, factoryName, styleProps: jsxStyleProps } = ctx.jsx
 
   const details = ctx.patterns.filterDetails(filters)
 
   return details.map((pattern) => {
     const { upperName, styleFnName, dashName, jsxName, props, blocklistType } = pattern
     const { description, jsxElement = 'div' } = pattern.config
 
+    const cssProps = match(jsxStyleProps)
+      .with('all', () => 'styleProps')
+      .with('minimal', () => '{ css: styleProps }')
```

Comment:
```suggestion
      .with('minimal', () => '{ css: mergeCss(styleProps, props.css) }')
```

We might need to merge. the incoming CSS from props with the style props

Reply:
you mean in the solid one, not preact then ?

Reply:
I mean in all frameworks (of course we need to test this) to make sure the use can still override the styles using props.css later on

Reply:
ah yes, I misread as `mergeProps` instead of `mergeCss`

I tried to make the changes, but doing so cause cyclic import issues I think, leading to `Maximum call stack size exceeded` with nuxt
`import { mergeCss } from '../css/css.mjs';`

not sure how to fix it since mergeCss needs the same `css` context